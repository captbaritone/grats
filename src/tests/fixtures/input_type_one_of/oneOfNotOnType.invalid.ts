/**
 * @gqlInput
 * @oneOf
 */
export class Greeting {}
