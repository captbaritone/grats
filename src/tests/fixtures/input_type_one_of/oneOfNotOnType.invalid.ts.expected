-----------------
INPUT
----------------- 
/**
 * @gqlInput
 * @oneOf
 */
export class Greeting {}

-----------------
OUTPUT
-----------------
src/tests/fixtures/input_type_one_of/oneOfNotOnType.invalid.ts:5:1 - error: Expected the type of a @gqlInput with @oneOf to be attached to a TypeScript union.

5 export class Greeting {}
  ~~~~~~~~~~~~~~~~~~~~~~~~
